cmd/cbuild/commands/root.go

```diff
@@ -161,9 +153,7 @@ func NewRootCmd() *cobra.Command {
 
 			fileExtension := filepath.Ext(inputFile)
 			var b builder.IBuilderInterface
-			if fileExtension == ".cprj" {
-				b = cproject.CprjBuilder{BuilderParams: params}
```

Comment:
Removing the `cprj` support here will break OTG acceptance tests because it requires changes in the build service.
Is it the right time to remove it or should we first keep it as undocumented feature until the build service will be updated accordingly?

Reply:
Thanks for pointing it out. I am planning to inform this change services and IDE team to see if they have some concerns or if they are ok to have these changes absorbed.

Reply:
Addressed [build CPRJ with root command](https://github.com/Open-CMSIS-Pack/cbuild/pull/66/commits/80820208e941c661ce61572da73b12a8bd082fe1)